docs/pages/index.js

```diff
@@ -150,21 +159,25 @@ export default function LandingPage(props) {
             </Container>
           </div>
           <div className={classes.social}>
-            <a
-              className="github-button"
-              href="https://github.com/mui-org/material-ui"
-              data-icon="octicon-star"
-              data-show-count="true"
-            >
-              Star
-            </a>
-            <a
-              className="twitter-follow-button"
-              href="https://twitter.com/@materialui"
-              data-show-screen-name="false"
-            >
-              Follow
-            </a>
+            <div className={classes.github}>
```

Comment:
Could you explain why this change was necessary and how it relates to the diamond sponsor?

Before this change the social buttons were centered. With this change they are a little bit off to the right. This doesn't seem intended since the rest of the landing page is centered.

master:
![sencha-before](https://user-images.githubusercontent.com/12292047/80909680-6c19e280-8d2a-11ea-8f0b-7cb724a46a1e.png)

pr:
![sencha-after](https://user-images.githubusercontent.com/12292047/80909684-6f14d300-8d2a-11ea-9fce-fab6b122d72f.png)

Reply:
Sure, I'm moving the change to #20877, the objective is to 1. avoid the layout jump, 2. make better centered (not mathematically, from a feeling perspective). If you feel it's less centered than before, I have failed 🙃.

Reply:
Done, I have already removed the changes related to 2.